en/complete_vehicles/px4_vision_kit.md

```diff
@@ -1,71 +1,97 @@
-It also covers the first flight, and how to get started with modifying the computer vision code.
+> **Note** This vehicle comes with no pre-installed software.
+  A pre-imaged USB stick that contains a reference implementation of the [PX4/Avoidance](../computer_vision/obstacle_avoidance.md) local planner software is provided by *Auterion*.
+   This software provides only a very basic example of what you can do with the PX4 Vision Autonomy Kit.
+   Developers can use the kit to try out other features provided by the [PX4 Avoidance](https://github.com/PX4/avoidance#obstacle-detection-and-avoidance) project, modify the existing code, or experiment with completely new computer vision-based functionality.
+
+The guide explains the minimal additional setup required to get the vehicle ready to fly (installing an RC system and battery). It also covers the first flight, and how to get started with modifying the computer vision code.
 
 ## Purchase
 
-Purchase link and more information about this kit [can be found here](http://www.holybro.com/product/px4-vision/).
+- [Holybro Website](https://shop.holybro.com/px4-vision_p1225.html?)
+
+## Px4 Vision Guide Content
+- [Warnings & Notifications](#warnings-and-notifications)
+- [What is Inside](#what-is-inside)
+- [What Else Do You Need](#what-else-do-you-need)
+- [First-time Setup](#first-time-setup)
+- [Fly the Drone with avoidance](#fly-the-drone-with-avoidance)
+- [Development using the Kit](#development-using-the-kit)
+- [Px4 Vision Carrier Board Pinouts](#px4-vision-carrier-board-pinouts)
+- [Other Development Resources](#other-development-resources)
+- [How to get Technical Support](#how-to-get-technical-support)
 
-## Warnings & Notifications
+## Warnings and Notifications
 
 1. The kit is intended for computer vision projects that use a forward-facing camera (it does not have downward or rear-facing depth cameras).
    Consequently it can’t be used (without modification) for testing [Safe Landing](../computer_vision/safe_landing.md), or other features that require a downward-facing camera.
-
 1. Obstacle avoidance in missions can only be tested when GPS is available (missions use GPS co-ordinates).
    Collision prevention can be tested in position mode provided there is a good position lock from either GPS or optical flow.
-
 1. The port labeled `USB1` may jam the GPS if used with a *USB3* peripheral (disable GPS-dependent functionality including missions).
    This is why the boot image is supplied on a *USB2.0* memory stick.
-
 1. The *UP Core* should only be powered using the battery (do not remove the *UP Core* power socket safety cover).
 
    ![Warning - do not connect power port](../../assets/hardware/px4_vision_devkit/warning_power_port.png)
 
-## Getting to Know Your DevKit
-
-![Schematic Overview](../../assets/hardware/px4_vision_devkit/px4_vision_schematic_overview.jpg)
+## What is Inside
+![Whats inside](../../assets/hardware/px4_vision_devkit/holybro_px4_vision_whats_inside_top.jpg)
```

Comment:
FWIW, and as I told Stone, these images are overly cluttered. Simply removing the graph paper underlay would make them much easier to read.